railties/lib/rails/generators/rails/scaffold_controller/templates/controller.rb

```diff
@@ -5,11 +5,23 @@
 <% module_namespacing do -%>
 class <%= controller_class_name %>Controller < ApplicationController
   before_action :set_<%= singular_table_name %>, only: [:show, :edit, :update, :destroy]
+  
+  <%- if collection_routing? -%>
+  # GET <%= route_url %>/1..10,17
+  def index
+    @<%= plural_table_name %> = <%= singular_table_name %>_ids ? <%= orm_class.find(class_name, "#{singular_table_name}_ids") %> : <%= orm_class.all(class_name) %>
+  end
 
+  # <%= singular_table_name.capitalize %> ids
+  def <%= singular_table_name %>_ids
```

Comment:
I see that is currently only called by the methods `index` and `update_many` from the same class.

Could you elaborate on why this method should be part of the public API? 
It doesn't look like an action that you might want to invoke from a route, but maybe there's something I'm missing.